core/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestTags.scala

```diff
@@ -9,4 +9,8 @@ object LeonardoTestTags {
   // intent: do not run in given execution context
   // this is also synonmous with 'azure test'
   object ExcludeFromJenkins extends Tag("ExcludeFromJenkins")
+
+  // intent: run in cron scheduled tests only, not at every PR commit
+  // example use case is for GPU testing that would otherwise run into resource allocation problems
+  object ScheduledTest extends Tag("ScheduledTest")
```

Comment:
a nit, but since is not truly 'scheduled', could you update this comment to say on merges to dev? and maybe switch up the naming?

Reply:
Will do, thanks!